New layout for mobile menu is ready.

YegorSan opened the pull request:
This PR is related to #2653 issue.

## Motivation
New layout for mobile menu was requested.

## Changelog
Created a new menu layout for mobile view.
***Not active state:**
---
*![Screenshot (1)](https://user-images.githubusercontent.com/10085945/64245985-a2d9f780-cf14-11e9-939e-bc29925208c5.png)
---
**Active state:**
---
![Screenshot (12)](https://user-images.githubusercontent.com/10085945/64246445-7a063200-cf15-11e9-94dd-5ca744709337.png)




## Checklist for your PR

<!--
  Ideally a PR has all of the checkmarks set.

  If something in this list is irrelevant to your PR, you should still set this
  checkmark indicating that you are sure it is dealt with (be that by irrelevance).

  If you don't set a checkmark (e. g. don't add a test for new functionality),
  you must be able to justify that.
-->

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

YegorSan commented:
@vbaranov  I have re-checked the files, should be better now. I am sorry, again VSCode turned on formatting for css files automatically.

YegorSan commented:
@vbaranov  Sorry for such a huge list of commits. Lot of tests didn't pass yesterday (ESLINT, test_geth_http_websocket, etc). I've fixed all the issues.

pashagonchar commented:
@vbaranov 
Menu items are not centred as in @pashagonchar screenshot in the issue #2653. I don't know should they be centred finally or not. @pashagonchar please check

Yes, I updated mockup in zeplin

![image](https://user-images.githubusercontent.com/16716926/64336833-9faf3c00-cfe6-11e9-9574-bcb4a77d2c86.png)

YegorSan commented:
@vbaranov  Hello. I am working on resolving mentioned above changes right now. Will try to resolve in the shortest terms.

vbaranov commented on apps/block_scout_web/assets/css/components/_navbar.scss:
```diff
@@ -258,4 +258,127 @@ $navbar-logo-width: auto !default;
+  margin-right: -7px !important;
+  padding-right: 0px;
+  height: 20px!important;
+  border-left: 1px solid #828ba0a1!important;
```
@YegorSan is the correct color hex here? It looks like it is invalid colour hex. It should be 3-byte hexadecimal code

vbaranov commented:
@YegorSan in addition, elements are still jumping https://www.loom.com/share/71119ba076624298aeace00fc02d2a5e

vbaranov commented:
@YegorSan in addition, I don't see dark mode switcher in the desktop mode anymore

![Screenshot 2019-09-06 at 15 37 22](https://user-images.githubusercontent.com/4341812/64428382-40bbf680-d0bc-11e9-9f5c-528b6017a5cb.png)

YegorSan commented:
@vbaranov  Hello. I've added fixed height for logo (was **max-height: 28px** now **height: 28px**). Jumping issue is resolved now.
https://www.loom.com/share/3b9d72fdf09f4954a2333d838c40d101
 I was not able to reproduce this issue locally as Blockscout main logo appearing by default, when I switched to Ethereum logo - issue was reproduced. Should be better now. Also changed color to 3-byte hexadecimal code, and dark mode switcher is appearing now in desktop mode. Please review. Thank's in advance.

vbaranov commented:
@YegorSan please resolve merging conflicts

YegorSan commented:
@vbaranov  Merging conflicts seems to be solved. Could you please review?  Thank's in advance.

vbaranov commented:
@YegorSan changelog entry...

YegorSan commented:
@vbaranov I am sorry. Changelog is updated.

YegorSan commented:
@vbaranov Hello Victor. I am sorry again, forgot to add separate JS onClick function for new button created. Now button is clickable and working as it expected.
